sherpa/sim/tests/test_sim.py

```diff
@@ -456,25 +456,26 @@ def test_mh(setup, caplog):
     # Note when the covariance changes; this is more just as a check
     # hence not a full check.
     #
-    diag = np.asarray([1.68857268e-03, 6.48507899e-01,
-                       9.22029508e-03, 1.88340111e-03,
-                       3.22767792e+00])
+    diag = np.asarray([1.68861299e-03, 6.48511837e-01,
+                       9.22037431e-03, 1.88340729e-03,
+                       3.22766923e+00])
     assert np.diag(cov) == pytest.approx(diag)
 
     # Check outfile; expect
     #     nfev statistic pl.gamma pl.ampl g1.fwhm g1.pos g1.ampl
     #
     row0 = np.asarray([0, -8975.692, 1.070194, 9.182625, 2.586208, 2.601620, 47.26266])
-    row19 = np.asarray([19, -8975.953, 1.069763, 9.219122,  2.570933, 2.586584,  47.36504])
-    row20 = row19.copy()
-    row20[0] = 20
+    row551 = np.asarray([551, -8975.953,  1.069770,  9.219028,
+                         2.570941,  2.586583,  47.36505])
+    row552 = row551.copy()
+    row552[0] = 552
 
     out.seek(0)
     fitvals = np.loadtxt(out)
-    assert fitvals.shape == (21, 7)
+    assert fitvals.shape == (553, 7)
```

Comment:
It is interesting that the fit is a lot more "inefficient" here. If we had infinite-time I'd suggest we look at whether we can get a "simpler" test case (one that requires less evaluations), but I am **not** suggesting it here.

However, the change does make sense (`NelderMead/Simplex` often requires more iterations than `LevMar`).